model/textparse/promparse_test.go

```diff
@@ -631,6 +639,35 @@ func BenchmarkParse(b *testing.B) {
 				}
 				_ = total
 			})
+			b.Run(parserName+"/created-lines/"+fn, func(b *testing.B) {
+				b.SetBytes(int64(len(buf) / promtestdataSampleCount))
+				b.ReportAllocs()
+				b.ResetTimer()
+
+				total := 0
+
+				st := labels.NewSymbolTable()
+				for i := 0; i < b.N; i += promtestdataSampleCount {
+					p := parser(buf, st)
+
+				Outer:
+					for i < b.N {
+						t, err := p.Next()
+						switch t {
+						case EntryInvalid:
+							if errors.Is(err, io.EOF) {
+								break Outer
+							}
+							b.Fatal(err)
+						case EntrySeries:
+							if parserName == "openmetrics/skip-ct" {
```

Comment:
I believe this conditional might make a small difference in the benchmark result, we're doing one extra operation that we probably won't do in the real code.

What do you think of moving this conditional up, before `b.Run(....`? I believe that's what you wanted to do, right? Just benchmark this if we're doing open-metrics

Reply:
hmm,
So i wanted both om parsers (with and without skipCT) to go through this test but I only call `p.createdTimestamp()` when skipCT is enabled. I believe thats what the goal is but we can't call that function when skipCT is disabled right?

or do we want to just test `p.createdTimestamp()` for om with skipct enabled?

Reply:
gonna close this one and restart conversation below